pkgs/development/tools/mold/default.nix

```diff
@@ -12,23 +12,15 @@
 
 stdenv.mkDerivation rec {
   pname = "mold";
-  version = "0.9.3";
+  version = "0.9.6";
 
   src = fetchFromGitHub {
     owner = "rui314";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256:1z9i8nvdl9h0zydh1gd9244q96n9x1gh5y90m71bghnh7nws0zmd";
+    sha256 = "sha256:0mj258fy8l4i23jd6ail0xrrq3das7lmrf1brrr1591ahx4vjj14";
   };
 
-  patches = [
-    # Intercept all invocations of ld, ld.gold or ld.lld
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Completely missed that, sorry.